tekton/pipeline-templates/params/fvt.yml.j2

```diff
@@ -1,13 +1,3 @@
-# FVT Image Container Registry
```

Comment:
See previous comment, I'm confused why these FVT specific params are being moved into common, that doesn't make sense to me.

Reply:
See comment above